content/en/platform/corda/4.12/community/getting-set-up.md

```diff
@@ -19,26 +19,24 @@ title: Getting set up for CorDapp development
 # Getting set up for CorDapp development
 
 There are four pieces of required software for CorDapp development:
-* Java 8 JDK
+* Java 17 JDK
 * IntelliJ IDEA
 * Git
 * Gradle
 
 {{< note >}} Visit the [platform support matrix]({{< relref "release-platform-support-matrix.md" >}}) to check the latest third-party application versions supported by Corda. {{< /note >}}
 
-## Installing the Java 8 JDK
+## Installing the Java 17 JDK
 
-Install the Java 8 JDK. Corda requires at least version 8u171, but do not currently support Java 9 or higher for this version of Corda.
+Install the Java 17 JDK. 
```

Comment:
Install the Java 17 JDK. Corda requires at least version 17.0.9, but does not currently support Java 18 or higher for this version of Corda.